### DISCUSSION TITLE
OKD 4.10 - Unable to install new operators

### DISCUSSION THREAD

**Discussion by gabrilabs75** [okd-project/okd#1429] [General]
Hi all,
trying to install new operators via okd console (from Red Hat marketplace rather than community) we always get the following error 

**Error getting YAML: YAMLException: unacceptable kind of an object to dump [object Undefined]**

**Version**
Server Version: 4.10.0-0.okd-2022-05-07-021833
Kubernetes Version: v1.23.5-rc.0.2068+b463d71c3d5ca3-dirty

Any suggestion?

Regards,
Gabriele

**Comment by vrutkovs** [comment]
Please attach (or upload to the public file sharing service) [must-gather](https://docs.okd.io/latest/support/gathering-cluster-data.html#support_gathering_data_gathering-cluster-data) archive

**Reply by gabrilabs75** [reply]
Hi,
we are not authorized to send this type of information as the cluster is dedicated to one of our clients.
In the meantime, can't you give us an initial indication?
If you need screenshots or output of commands please let me know.

Regards,
Gabriele

**Comment by JaimeMagiera** [comment]
OKD is community-supported, which means we are all volunteers. For community members to volunteer their time effectively and efficiently, users need to provide must-gather archives. If there are concerns about sensitive data, you could conceivably scrub or replace that data. A web search might yield some ideas and tools to do this. 

I cannot vouch for this project, but it is one option. 

https://github.com/openshift/must-gather-clean

**Reply by gabrilabs75** [reply]
hi guys,
we managed to solve the problem.

Regards,
Gabriele

**Reply by minhtuanvu** [reply]
Hi @gabrilabs75, how did you fix this, I also got this error.
